### DISCUSSION TITLE
FrankenPHP & Dynatrace

### DISCUSSION THREAD

**Discussion by ARacamond** [php/frankenphp#1232] [Q&A]
Hi,

I am running a server with FrankenPHP on a Kubernetes stack.
Previously, I had monitoring set up via Dynatrace using Nginx and PHP-FPM. However, since migrating to FrankenPHP, Dynatrace no longer provides any data.

Is there anything specific that needs to be done? Could FrankenPHP be missing something required for monitoring with Dynatrace?
Has anyone encountered a similar use case? At this point, I am unsure how to enable monitoring effectively.

Thank you in advance for your assistance.

**Comment by efedevd** [comment]


**Reply by pierre-cba** [reply]
@efedevd your response seems to be an AI response ah ah ; 

on our team, we have a kubernetes stack too , and we monitor sucessfully other application like php fpm , nginx , java spring, and wildfly. 
so our oneAgent it's ok

only our new pod with franken php seems to be not working anymore with the same configuration.

if you have any idea ? 

Cordialement Pierre.

**Comment by withinboredom** [comment]
Their extension (I assume it is a php extension) is not open source, so you'll need to file a bug ticket with them to investigate the issue. There's not much we can do from our end; but if they have any questions or help, we'll be happy to assist.

**Comment by pierre-cba** [comment]
Hello,
for information and for people who want Dynatrace on FrankenPHP, there is a thread in the Dynatrace product ideas section. The more votes it gets, the higher priority the topic will receive, so don't hesitate to check it out.

https://community.dynatrace.com/t5/Product-ideas/Support-FrankenPHP-Native-support-or-custom-extension/idi-p/265336

Best regards,
Pierre.
